XamlControlsGallery/ControlPages/TabViewPage.xaml

```diff
@@ -300,7 +300,7 @@
             </local:ControlExample.Xaml>
         </local:ControlExample>
 
-        <local:ControlExample HeaderText="Complete TabView windowing sample">
+        <local:ControlExample HeaderText="Complete TabView windowing sample" MinimumUniversalAPIContract="8">
```

Comment:
I'm confused. Doesn't specifying minimum contract version for the ControlExample hide the entire example for contracts  < 8? That's certainly not what your screenshot is showing, and I haven't tried this on an older build. So maybe my understanding of how MinimumUniversalAPIContract works is wrong.

But where is the error text shown in your screenshot?

Reply:
Specifying the minimum contract version doesn't hide/disable the control, no. For example, the editable ComboBox sample just displays as a regular ol' ComboBox (that isn't editable) on lower versions of the SDK. We deliberately decided not to fully hide samples even on lower versions of the SDK if there is reasonable fallback behavior. 

The error text in the screenshot above is the red text above the button which is provided by the MinumumUniversalAPIContract API. There is also a comment in the code.